config/README.rst

```diff
@@ -98,7 +98,7 @@ Preparation
 
 The script needs a ``venv`` with some packages installed::
 
-    $ python3.11 -m venv .
+    $ python3 -m venv .
```

Comment:
This change is not correct: The code in this repository requires to use Python 3.11+.

Reply:
You changed that from Python 3.8 to 3.11 in https://github.com/zopefoundation/meta/commit/1ef8b9b8ae8311a7e79edb13d5086aed20bda382 but I can't see anything in that change set that requires 3.11+. I know that #209 would require 3.11, but that's not merged.